qmri_mtsat/derivatives/qMRLab/sub-01/fmap/sub-01_TB1map.json

```diff
@@ -1,7 +1,7 @@
 {
     "RawSources": [
-      "sub-01/fmap/sub-01_flip-01_TB1DAM.nii.gz",
-      "sub-01/fmap/sub-01_flip-02_TB1DAM.nii.gz"
+      "bids:source:sub-01/sub-01/fmap/sub-01_flip-01_TB1DAM.nii.gz",
+      "bids:source:sub-01/sub-01/fmap/sub-01_flip-02_TB1DAM.nii.gz"
```

Comment:
I think `RawSources` does not permit BIDS URIs, only dataset relative links. Also, that field is deprecated in favor of `Sources` (which does permit BIDS URIs)

https://bids-specification.readthedocs.io/en/latest/glossary.html#rawsources-metadata

I think we should either just use relative links here, OR turn the `RawSources` here into `Sources`